templates/repo/signature_badge.tmpl

```diff
@@ -0,0 +1,91 @@
+{{$base_classes := "show-modal tw-cursor-pointer tw-rounded-full tw-border tw-px-2 tw-py-0"}}
+{{$verified_colors := "tw-border-success-border tw-bg-success-bg tw-text-success-text"}}
+{{$unverified_colors := "tw-border-warning-border tw-bg-warning-bg tw-text-warning-text"}}
+{{if .verification.Verified}}
+    {{if eq .verification.TrustStatus "trusted"}}
+        <div class="{{$base_classes}} {{$verified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Verified</div>
+    {{else if eq .verification.TrustStatus "untrusted"}}
+        <div class="{{$base_classes}} {{$unverified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Unverified</div>
+    {{else}}
+        <div class="{{$base_classes}} {{$unverified_colors}}" data-modal="#{{.shortsha}}-verification-modal">Unverified</div>
```

Comment:
The else if here is obsolete, isn't it?

Reply:
Depends on backend code. Can `.verification.Verified` be true while `.verification.TrustStatus = "untrusted"`? If not, then yes, it is dead code.

Reply:
From how I read the code, it's basically the following situation, because I'm adding the exact same div in line 8 and line 10:
```
if (condition) {
  do_x();
} else if (condition2) {
  do_y();
} else {
  do_y();
}
```
So no matter, if the verification status is untrusted or anything else, it will render the exact same div, or is there something I'm missing here?

Reply:
You are right, these two if branches output the same content so it can be reduced to a single `else` branch.